pkg/sypgp/sypgp.go

```diff
@@ -718,6 +718,10 @@ func RecryptKey(k *openpgp.Entity) error {
 	return nil
 }
 
+// ReformatGPGExportedFile reformats a reader that parses a set of GPG keys
+// based on the standard format (human readable) to a reader that follows the
+// machine readable index format. The reformat allows us to easily parse the
+// keys.
```

Comment:
great description :+1: would also be useful to add at the end the url to the format that OpenPGP uses: https://tools.ietf.org/html/draft-shaw-openpgp-hkp-00#section-5.2

Reply:
Good idea. FYI, i included an empty line before the link so it does not appear in the doc. I think this is mainly meant for people look at the code rather than developers who are trying to use the function. Does it make sense?

Reply:
sure no problem :+1: lgtm